market_env/environment.py

```diff
@@ -1,16 +1,20 @@
 # pylint: disable=missing-module-docstring
```

Comment:
you should add doc strings wherever possible to improve code understandability instead of disabling warning